lib/puppet/provider/elasticsearch_plugin/plugin.rb

```diff
@@ -90,7 +98,12 @@ def destroy
   def es_version
     return @es_version if @es_version
     begin
-      version = es('-v') # ES 1.x
+      os = Facter['osfamily'].value
+      if os == 'OpenBSD'
+        version = pkg_info('elasticsearch').lines.first
```

Comment:
That's weird it doesn't happen on the debian/ubuntu/centos based machines.
elasticsearch binary should be handling the elasticsearch.in.sh automatically.